recipe/meta.yaml

```diff
@@ -27,13 +26,16 @@ requirements:
     - wheel
   run:
     - python
-    - setuptools
-    - wheel
     - packaging
 
 test:
+  requires:
+    - pytest
+    - pip
   imports:
     - mgrs
+  commands:
+    - pip check
 
 about:
```

Comment:
Add a dev/doc_url and a description here

Reply:
Changed. Thanks :)